tasks/pre.yml

```diff
@@ -66,17 +66,66 @@
   tags:
     - fact
 
-- name: redhat family epel installation
+- name: RedHat family (Centos etc..) EPEL installation
```

Comment:
not really a necessary change, since redhat family imples centos etc

Reply:
I can make different CentoOS and real RHEL8 here. What works on CentOS, is not functional on the real RHEL .. Different repos, etc...

Reply:
Well, there is a difference but it still the same distribution family, especially now since CentOS is becoming CentOS Stream.